eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Transition.hs

```diff
@@ -52,6 +53,6 @@ toAlonzoTransitionConfigPairs = toKeyValuePairs
 
 instance FromJSON (TransitionConfig AlonzoEra) where
   parseJSON = withObject "AlonzoTransitionConfig" $ \o -> do
-    pc <- parseJSON (Object o)
-    ag <- o .: "alonzo"
-    pure $ mkTransitionConfig pc ag
+    prevTransitionConfig :: TransitionConfig MaryEra <- parseJSON (Object o)
+    alonzoGenesis :: AlonzoGenesis <- o .: "alonzo"
+    pure $ mkTransitionConfig alonzoGenesis prevTransitionConfig
```

Comment:
Oh we were parsing them the wrong way around?

Reply:
Yep, that is exactly it.